src/Workspaces/CoreTest/CodeCleanup/ReduceTokenTests.cs

```diff
@@ -1965,29 +1965,37 @@ End Module
             var code = @"[|
 Module Module1
     Sub Main()
-        Dim sMax As Short = &B101
-        Dim usMax As UShort = &B00100
-        Dim iMax As Integer = &B00100100
-        Dim uiMax As UInteger = &B001001100110
-        Dim lMax As Long = &B001001100110
-        Dim ulMax As ULong = &B001001100110
-        Dim z As Long = &B001001100110
-        Dim x As Long =&B001001100110
+        ' signed
+        Dim a As Short = &B101
+        Dim b As Integer = &B00100100
+        Dim c As Long = &B001001100110
+
+        Dim d As UShort = &B00100
+        Dim e As UInteger = &B001001100110
+        Dim f As ULong = &B001001100110
+
+        Dim g As Short = -&B101
+        Dim h As Integer = -&B00100100
+        Dim i As Long = -&B001001100110
```

Comment:
These are good. However, what I meant is to test binary literals that themselves represent negative numbers. Also, it looks like we are not testing Byte/SByte.